master/internal/api_tasks.go

```diff
@@ -414,6 +416,10 @@ func (a *apiServer) PostTaskLogs(
 		return nil, fmt.Errorf("adding task logs to task log backend: %w", err)
 	}
 
+	if err := webhooks.ScanLogs(ctx, logs); err != nil {
```

Comment:
i think it is strange that the webhooks package scans the logs rather than the logpattern package scanning the logs and the API layer deciding to write a webhook event as a result of that scan.

Reply:
talked offline and decided this is okay because it is so bolted on it is easy to remove or change